src/JuMP.jl

```diff
@@ -19,30 +19,20 @@ module JuMP
 
 import Base.Meta: isexpr, quot
 import LinearAlgebra
-import MathOptInterface
+import MathOptInterface as MOI
 import MutableArithmetics
 import OrderedCollections
 import OrderedCollections: OrderedDict
 import Printf
 import SparseArrays
 
+# We can't use import MutableArithmetics as _MA because of a bug in MA.
```

Comment:
This will be fixed by https://github.com/jump-dev/MutableArithmetics.jl/pull/199